product_docs/docs/pem/9/considerations/pem_pgbouncer/configuring_pgBouncer.mdx

```diff
@@ -69,23 +72,33 @@ You must configure PgBouncer to work with the PEM database server. This example
     server_idle_timeout = 60
     ```
 
-4.  Create and update the `/etc/edb/pgbouncer1.9/userlist.txt` authentication file for PgBouncer:
+!!! Note
+    For more information on `auth_user` see [authentication settings](https://www.pgbouncer.org/config.html#authentication-settings).
+
+4.  Create and update the `/etc/edb/pgbouncer<x.x>/userlist.txt` authentication file for PgBouncer:
 
     ```sql
+    ## Connect to pem database as a superuser, 
+    ## create the userslist.txt file and add 
+    ## username and their password list in CSV format
+    $ psql -p 5444 -d pem -U enterprisedb
+    
     pem=# COPY (
     SELECT 'pgbouncer'::TEXT, 'pgbouncer_password'
     UNION ALL
     SELECT 'pem_admin1'::TEXT, 'pem_admin1_password'
-    TO '/etc/edb/pgbouncer1.9/userlist.txt'
+    TO '/etc/edb/pgbouncer<x.x>/userlist.txt'
     WITH (FORMAT CSV, DELIMITER ' ', FORCE_QUOTE *);
-
+    __OUTPUT__
     COPY 2
     ```
 
+    This creates `/etc/edb/pgbouncer<x.x>/userlist.txt` file and adds the username and password list.
+
     !!! Note
         A superuser can't invoke the PEM authentication query function `pem.get_proxy_auth(text)`. If the pem_admin user is a superuser, you must add the password to the authentication file (`enterprisedb` in the example).
```

Comment:
It isn't completely clear how these two sentences relate. I think what we are trying to say is:
```suggestion
        If the pem_admin user is a superuser, you must add the password to the authentication file (`enterprisedb` in the example). This allows the pem_admin user to invoke the PEM authentication query function `pem.get_proxy_auth(text)`.
```